packages/api/src/command/medical/cq-directory/rebuild-cq-directory.ts

```diff
@@ -0,0 +1,105 @@
+import { Carequality } from "@metriport/carequality-sdk/client/carequality";
+import { sleep } from "@metriport/core/util/sleep";
+import { QueryTypes, Sequelize } from "sequelize";
+import { Config } from "../../../shared/config";
+import { capture } from "../../../shared/notifications";
+import { bulkInsertCQDirectoryEntries } from "./create-cq-directory-entry";
+import { parseCQDirectoryEntries } from "./parse-cq-directory-entry";
+
+const BATCH_SIZE = 1000;
```

Comment:
It would be nice to get this from the CQ SDK. Imagine we change this on the SDK to 1,200 and forget to update this here. It will make it so we only bring the first batch from CQ when executing `rebuildCQDirectory`.

Or we pass this to `listOrganizations` so we're defining the page size when querying.

Reply:
Good point!